Api/Core/OAuth2/Repository/ScopeRepository.php

```diff
@@ -0,0 +1,44 @@
+<?php
+namespace Api\Core\OAuth2\Repository;
```

Comment:
This should be in the lib folder. Why are we doing this?

Reply:
I've made only a working example. We can place it wherever we want after we discussed.